pywbem/cim_operations.py

```diff
@@ -861,6 +890,100 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_server_response_time = None
 
     @property
+    def operation_recorder(self):
+        """
+        This is a writeable property; setting this property will cause the
+        defined recorder to be activated.
+
+        Effective pywbem 0.10.0 operation_recorder became a property rather
+        than an attribute. The getter operation returns TODO
```

Comment:
It returns the recorder that was last written to it, just like any other writeable property. I'm not sure we need to mention that.
What is important to mention though is that setting this property will *add* the recorder, not just replace the single recorder.

Reply:
Done.